inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -711,6 +712,7 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         manager.register_pass<ngraph::pass::ConvertOpSet3ToOpSet2>();
         manager.register_pass<ngraph::pass::ConvertOpSet2ToOpSet1>();
         manager.register_pass<ngraph::pass::ConvertOpSet1ToLegacy>();
+        manager.register_pass<ngraph::pass::LSTMCellDecomposition>();
```

Comment:
Maybe there is a sense to put legacy transformations under if condition and don't call them for graph based functions where we have a similar ngraph pass.